clippy_lints/src/suspicious_xor_used_as_pow.rs

```diff
@@ -34,21 +36,20 @@ impl LateLintPass<'_> for ConfusingXorAndPow {
             left.span.ctxt() == right.span.ctxt() &&
```

Comment:
Since this was your first lint, I'll also be a bit more nitpicky :wink: 

Can you fix the formatting, to have the && before the condition and not at the end of the line and also remove the tab character in line 34?